cranelift-codegen/src/isa/x86/enc_tables.rs

```diff
@@ -895,6 +895,80 @@ fn expand_fcvt_to_uint_sat(
     cfg.recompute_ebb(pos.func, done);
 }
 
+/// Convert shuffle instructions.
+fn convert_shuffle(
```

Comment:
A suggestion for a follow-up: whenever you think is the right time, it might be nice to move all the SIMD legalizations to their own file in the isa/x86 directory.

Reply:
Sounds good, I'll do this in one of the next few PRs.